packages/noya-state/src/reducers/pointReducer.ts

```diff
@@ -114,6 +121,42 @@ export function pointReducer(
         );
       });
     }
+
+    case 'setLinePointX':
+    case 'setLinePointY': {
```

Comment:
There might be an opportunity to combine these with `setPointX`/`setPointY` by passing a `SelectedPointLists`. I'm not sure if it'd be an improvement... but I suspect it would